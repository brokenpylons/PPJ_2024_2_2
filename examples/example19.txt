var result = 0;
for (var i = 1, 5) {
    result = result + i;
}

for (var i = 1, 3) {
    result = result * i;
}

print(result);
