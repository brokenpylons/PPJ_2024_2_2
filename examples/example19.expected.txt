variable("x") plus("+") variable("y") 
